website_sale_empty_cart/views/cart_templates.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <template id="total_empty_cart" inherit_id="payment.express_checkout">
+        <xpath expr="//form[@name='o_payment_express_checkout_form']" position="before">
+            <div class=" d-flex flex-column mb-5 mb-lg-0 pt-4">
```

Comment:
The div should not be drawn if the option is not enabled, as otherwise unnecessary elements are added to the view.
t-if="website.cart_empty_button" --> move it to the div